ui/imports/utils/Utils.qml

```diff
@@ -869,7 +869,7 @@ QtObject {
     }
 
     function isBase64DataUrl(str) {
-        return /^data:.*;base64,/.test(str);
+        return globalUtilsInst.isBase64DataUrl(str)
```

Comment:
Please check this comment: https://github.com/status-im/status-desktop/pull/18727/files#r2303224862

Reply:
I totally get your point. I'll improve it for the master version. For the release version I'll just leave it as is just to be able to cut sooner